conf.d/mysql.yaml.example

```diff
@@ -13,6 +13,9 @@ instances:
     # options:               # Optional
     #   replication: 0
     #   galera_cluster: 1
+    #   extra_status_metrics: 0
```

Comment:
We just use proper `no` or `false` statements here

You can then use the _is_affirmative function to convert that into booleans